pkg/controller/provider/web/vsphere/vm.go

```diff
@@ -56,6 +56,8 @@ func (h VMHandler) List(ctx *gin.Context) {
 			ctx.Status(http.StatusInternalServerError)
 		}
 	}()
+
+	h.Detail = model.MaxDetail
```

Comment:
Ah one more note, please add note that we need the max detail for the IsTemplate